### PR TITLE
Fix type definitions

### PR DESCRIPTION
**Pull request by webberig**
- Fixed `package.json` so definition files is not being ignored during npm publish.
- Fix to the type definition so property `container` is not required when passing options to the leaflet plugin

### CODE REVIEW

File: leaflet-maplibre-gl.d.ts [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import * as L from 'leaflet';
 import { Map, MapboxOptions } from 'maplibre-gl';
 
+
 declare module 'leaflet' {
+    type LeafletMaplibreGLOptions = Omit<MapboxOptions, "container">;
```
Comment (wipfli) [inline leaflet-maplibre-gl.d.ts:6]: Can you describe why `container` is problematic?

**Review by wipfli — COMMENTED** [review]
Files part looks good to me.

File: leaflet-maplibre-gl.d.ts (hunk 1 above)
Comment (webberig) [reply leaflet-maplibre-gl.d.ts:6]: This plugin creates its own container (being a Leaflet layer) and passes this container to the Maplibre-GL instance.
See: https://github.com/maplibre/maplibre-gl-leaflet/blob/main/leaflet-maplibre-gl.js#L120

Passing a `container` property would simply be ignored. However, the type (MapboxOptions) defines `container` as a required property which is not the case when passing options to this Leaflet plugin. This is why the type of the options you can pass must change.

**Review by wipfli — APPROVED** [review]

**Comment by jacoux** [comment]
in maplibre-gl": ">=2.0.0"MapboxOptions is changed in MapOptions. 
Might be good to keep this in mind when merging.

**Pull request merged**